lib/resources/r4/questionnaire_response.yaml

```diff
@@ -23,4 +23,168 @@ fields:
     example: |
       {
         "id": "SH-12508041"
-      }
\ No newline at end of file
+      }
```

Comment:
i think we need to include the two extensions here?

Reply:
I also added a not on choice answer to tell them that it's required when answer is coding and should not be used when answer is quantity. Since it says it's not required, I thought some clarification would be good: 78becfe9c9f89da6de94c68d214dba6c408728c6 & https://github.com/cerner/fhir.cerner.com/pull/669/commits/fc013ad5d295a324502c0853abce974925c939f8

Reply:
In the overview returned fields? yeah i think so, one under item and one under item.item

Reply:
hmm personally, i dont like the second part, and i think ur first part is enough, about corresponding to a choice of open choice question. 
Bc sometimes they should have it for just a value string and sometimes not so how would we say that? and the logic isnt 'bc this is an answer quantity, i shouldnt have the extension" the logic should be 'this is answering for a quantity question so shouldnt put it"
Thoughts though?

Reply:
Yeah I see your point, thinking on it again I agree. I I removed all of it here: https://github.com/cerner/fhir.cerner.com/pull/669/commits/5b11f976693b257755285302b19f4c1c7875e3df